ut-user/index.js

```diff
@@ -16,12 +20,28 @@ function getCrypt(cryptKey) {
     return crypt;
 }
 
+function getIPAddress() {
+  var interfaces = os.networkInterfaces();
+  for (var devName in interfaces) {
+    var iface = interfaces[devName];
+
+    for (var i = 0; i < iface.length; i++) {
+      var alias = iface[i];
+      if (alias.family === 'IPv4' && alias.address !== '127.0.0.1' && !alias.internal)
```

Comment:
Can we use the ip address to which the Hapi server connection object bounds to ? This IP address is available in the server.info.address property.